### ISSUE TITLE
Error on CentOS 7 - NetworkManager is not running

### ISSUE THREAD

**Issue by fwalker** [FujitsuEnablingSoftwareTechnologyGmbH/devstack-vagrant#2]
Starting the provisioning as described in the readme, leads to the following error:

```
==> devstack: Waiting for SSH to become available...
    devstack: 
    devstack: Vagrant insecure key detected. Vagrant will automatically replace
    devstack: this with a newly generated keypair for better security.
    devstack: 
    devstack: Inserting generated public key within guest...
    devstack: Removing insecure key from the guest if it's present...
    devstack: Key inserted! Disconnecting and reconnecting using new SSH key...
==> devstack: Configuring and enabling network interfaces...
The following SSH command responded with a non-zero exit status.
Vagrant assumes that this means the command failed!
! which nmcli >/dev/null 2>&1 || nmcli c reload eth1
Stdout from the command:
Stderr from the command:
Error: NetworkManager is not running.
```

Potentially, it is required that my NetworkManager is running (which is deactivated)?

**Comment by zreigz** [comment]
Theoretically your local network has been changed and you should execute `vagrant destroy` before `vagrant up`

**Comment by zreigz** [comment]
I found bug which is related to your problem: https://github.com/mitchellh/vagrant/issues/6487
What is your vagrant version ?
I use Vagrant 1.8.1 and this problem doesn't exist

**Comment by cheld** [comment]
I just created OpenStack with DevstackVagrant and had no problems. Currently the machine is used by Atanas. He recreated OpenStack. We can ask him when he is in office

**Comment by fwalker** [comment]
I had 1.7.1 installed. After the update to 1.8.1 (and reinstalling the vatrant-libvirt plugin) it's working well. Maybe this could be reflected in the readme ^^/

**Comment by cheld** [comment]
updated docs

**Comment by fwalker** [comment]
Spotted 2 placed in the doc wher it'S still <> 1.8.1
`sudo dpkg -i vagrant_1.7.2_x86_64.deb`
and
`sudo yum install vagrant_1.7.3_x86_64.rpm`

**Comment by ohtake** [comment]
> Spotted 2 placed in the doc wher it'S still <> 1.8.1

Fixed in 27d45ebd22cef5946bee97f7d4bc02cca2519bf6.
